src/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, BigInt, Bytes } from "@graphprotocol/graph-ts";
+
+export function loadOrCreateEntity<T extends Entity>(
+  id: Bytes,
+  loadFn: (id: Bytes) => T | null,
+  createFn: (id: Bytes) => T
+): T {
+  let entity = loadFn(id);
+  if (entity == null) {
+    entity = createFn(id);
+  }
+  return entity;
+}
+
+export const DEFAULT_BIGINT = BigInt.zero();
+export const DEFAULT_BYTES = Bytes.empty();
+export const CONTRACT_CONFIG = Bytes.fromUTF8("default");
```

Comment:
I'm not particularly fond of this pattern. I'd rather see default values in the schema so that the generated constructor default to these